TeamCode/src/main/java/org/firstinspires/ftc/teamcode/ultimategoal/modules/OdometryModule.java

```diff
@@ -67,8 +76,12 @@ public synchronized void update() {
         return data;
     }
 
-    public void fileDump(){
-        robot.fileDump.addData(concat("1"," ","odometry.txt"), concat(worldX," ",worldY," ",worldAngleRad));
+    public String getFileName() {
+        return startTime + "_odometry.txt";
```

Comment:
I think for every file dump they should be in the same subdirectory titled the startTime. Inside that subdirectory are odometry.txt, shooter.txt, etc.